roles/operator_bundle_scorecard_tests/files/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -3,32 +3,37 @@
 set -o nounset
 set -o errexit
 set -o pipefail
+set -x
 
 # This value serves as a default when the parameters are not set, which should
 # only happen in rehearsals. Production jobs should always set the OO_* variable.
 REHEARSAL_BUNDLE="brew.registry.redhat.io/rh-osbs-stage/e2e-e2e-test-operator-bundle-container:8.0-3"
 OO_BUNDLE="${OO_BUNDLE:-$REHEARSAL_BUNDLE}"
+OPENSHIFT_AUTH="${OPENSHIFT_AUTH:-/var/run/brew-pullsecret/.dockerconfigjson}"
+SCORECARD_CONFIG="${SCORECARD_CONFIG:-/tmp/config/scorecard-basic-config.yml}"
 
 # Steps for running the basic operator-sdk scorecard test
 # Expects the standard Prow environment variables to be set and
 # the brew proxy registry credentials to be mounted
 
 NAMESPACE=$(grep "install_namespace:" "${SHARED_DIR}"/oo_deployment_details.yaml | cut -d ':' -f2 | xargs)
 
-OPERATOR_DIR="${ARTIFACT_DIR}/test-operator-basic"
+pushd "${ARTIFACT_DIR}"
```

Comment:
There is missing popd counterpart

Reply:
It's on line 31

Reply:
That is not true. The line 31 is counterpart for the line 28